configs/sst_virtualization_advanced-virt.yaml

```diff
@@ -0,0 +1,46 @@
+document: feedback-pipeline-workload
+version: 1
+data:
+  name: Advanced Virtualization on RHEL - for Red Hat Layered Products
+  description: Userspace agents and tools that enable advanced virtualization capabilities on RHEL
+  maintainer: sst_virtualization_advanced
+
+  modules_enable:
+  - virt:8.4
+
+  packages:
+  - hivex
+  - kata-agent
+  - kata-osbuilder
+  - kata-runtime
+  - kata-shim
+  - libguestfs
+  - libguestfs-winsupport
+  - libiscsi
+  - libnbd
```

Comment:
I would really like `libnbd` to appear in AppStream in RHEL 9.  It is in RHEL 8 virt module but that was a mistake.